extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -337,18 +346,22 @@ export const AzureProvisionDialog: React.FC = () => {
     setPage(page);
```

Comment:
Isn't it great how I added this function so you had a place to shove telemetry? :)

Reply:
FOR REALS.